src/app/features/mondomarket/index.js

```
@@ -236,6 +270,16 @@ const mondoMarketSlice = createSlice({
       .addCase(searchMondoItem.rejected, (state, action) => {
         state.isSearching = false;
         state.error = action?.payload;
+      })
+      .addCase(toggleAlert.pending, (state) => {
+        state.isLoading = true;
+      })
+      .addCase(toggleAlert.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)